appengine/standard/iap/js/poll.js

```diff
@@ -1,11 +1,11 @@
-// Copyright Google Inc.
-//
+// Copyright 2024 Google Inc.
```

Comment:
the year should be from the original file date here, in this case it is 2017, and it needs to be Google LLC not Google Inc.